zfs-dkms/PKGBUILD

```diff
@@ -17,15 +17,18 @@ provides+=('zfs')
 replaces=('spl-dkms')
 source=("https://github.com/zfsonlinux/zfs/releases/download/zfs-${pkgver}/zfs-${pkgver}.tar.gz"{,.asc}
         "https://github.com/openzfs/zfs/commit/2fcab8795c7c493845bfa277d44bc443802000b8.patch"
-        "0001-only-build-the-module-in-dkms.conf.patch")
+        "0001-only-build-the-module-in-dkms.conf.patch"
+        "0002-fix-5.6-build.patch")
```

Comment:
Please reverse the order of the patches, and add fix-5.6-build.patch after 2fcab8795c7c493845bfa277d44bc443802000b8.patch but before only-build-the-module-in-dkms.conf.patch -- this reduces the number of line changes due to the closing `)` and groups the new patch with the 5.5 patch.